{"name": "一键呼救通话记录获取", "status": "broken", "statusDetails": {"message": "TypeError: 'NoneType' object is not subscriptable", "trace": ">   ???\nE   TypeError: 'NoneType' object is not subscriptable\n\nD:\\work\\jishiyu\\Interface_Test\\test_case\\opsPC\\conftest.py:10: TypeError"}, "start": 1692597242916, "stop": 1692597242916, "uuid": "4296df29-4ca5-4ef5-95a1-d068352131c6", "historyId": "e98543b432b2c6de3d4976978aeda2d2", "testCaseId": "4c3c898f7ab9936c751e31bf0b10a250", "fullName": "opsPC.test_emergency#test_elevator_call_record_info_001", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "应急处置"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "14852-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}